components/redmine/redmine.app.mjs

```diff
@@ -1,11 +1,231 @@
+import {
+  axios, ConfigurationError,
+} from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+import utils from "./common/utils.mjs";
+
 export default {
   type: "app",
   app: "redmine",
-  propDefinitions: {},
+  propDefinitions: {
+    projectId: {
+      type: "integer",
+      label: "Project ID",
+      description: "The ID of the project",
+      async options() {
+        const response = await this.listProjects();
+        console.log(response);
+        const { projects } = response;
+        return projects.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    trackerId: {
+      type: "integer",
+      label: "Tracker ID",
+      description: "The ID of the tracker",
+      async options() {
+        const { trackers } = await this.listTrackers();
+        return trackers.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    statusId: {
+      type: "integer",
+      label: "Status ID",
+      description: "The ID of the status",
+      async options() {
+        const { issue_statuses: statuses } = await this.listIssueStatuses();
+        return statuses.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    priorityId: {
+      type: "integer",
+      label: "Priority ID",
+      description: "The ID of the priority",
+      async options() {
+        const { issue_priorities: priorities } = await this.listIssuePriorities();
+        return priorities.map(({
+          id: value, name: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    userId: {
+      type: "integer",
+      label: "User ID",
+      description: "The ID of the user",
+      async options() {
+        const { users } = await this.listUsers();
+        return users.map(({
+          id: value, login: label,
+        }) => ({
+          label,
+          value,
+        }));
+      },
+    },
+    issueId: {
+      type: "integer",
+      label: "Issue ID",
+      description: "The ID of the issue",
+    },
```

Comment:
I don't see the prop `issueId` being used anywhere. It can be removed if it's not needed.
```suggestion
```